### PR TITLE
Bug 1292270 - Pass a User object down to JobManager.update_after_verification.

### PR DESCRIPTION
**Pull request by jgraham**
This is required to create the BugJobMap instance in the post-datasource world.

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/treeherder/1760)

### CODE REVIEW

File: tests/settings.py [hunk 1]
```diff
@@ -20,3 +20,5 @@
 ELASTIC_SEARCH.update({
     "index_prefix": "test",
 })
+
+AUTOCLASSIFY_JOBS = True
```
Comment (edmorley) [inline tests/settings.py]: I think we should make the default in config/settings.py be True instead perhaps?

**Comment by camd** [comment]
Reviewed 4 of 4 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[tests/settings.py, line 24 \[r1\]](https://reviewable.io:443/reviews/mozilla/treeherder/1760#-KOMMQcMAn7CNj-v1ZRg-r1-24:-KOMeENcBGUKkR4Ks1fD:2060362700) ([raw file](https://github.com/mozilla/treeherder/blob/fa41a91aa0c9511b4b5106d522e3b32f9ab36f30/tests/settings.py#L24)):*
<details><summary><i>Previously, edmorley (Ed Morley) wrote…</i></summary>
> I think we should make the default in config/settings.py be True instead perhaps?
</details>
good point. I agree.  :)

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/mozilla/treeherder/1760)*

**Comment by camd** [comment]
Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/mozilla/treeherder/1760)*

**Comment by camd** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[tests/settings.py, line 24 \[r1\]](https://reviewable.io:443/reviews/mozilla/treeherder/1760#-KOMMQcMAn7CNj-v1ZRg-r1-24:-KON1e18lZTsBz0y4Fu4:160415369) ([raw file](https://github.com/mozilla/treeherder/blob/fa41a91aa0c9511b4b5106d522e3b32f9ab36f30/tests/settings.py#L24)):*
<details><summary><i>Previously, camd wrote…</i></summary>
> good point. I agree.  :)
</details>
ok

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/mozilla/treeherder/1760)*

**Comment by camd** [comment]
:lgtm:

---

Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/mozilla/treeherder/1760#-:-KON1vZhqzQmIZWG_HDH:1417088721)*

File: tests/settings.py (hunk 1 above)
Comment (edmorley) [inline tests/settings.py]: This can be removed, now that the default was changed

**Pull request merged**